pkg/cli/admin/policy/modify_scc.go

```diff
@@ -231,105 +231,63 @@ func (o *SCCModificationOptions) CompleteGroups(f kcmdutil.Factory, cmd *cobra.C
 }
 
 func (o *SCCModificationOptions) AddSCC() error {
-	scc, err := o.SCCInterface.Get(context.TODO(), o.SCCName, metav1.GetOptions{})
-	if err != nil {
-		return err
-	}
-
-	users, groups := stringSubjectsFor(o.DefaultSubjectNamespace, o.Subjects)
-	usersToAdd, _ := diff(users, scc.Users)
-	groupsToAdd, _ := diff(groups, scc.Groups)
-
-	scc.Users = append(scc.Users, usersToAdd...)
-	scc.Groups = append(scc.Groups, groupsToAdd...)
-
-	message := successMessage(true, o.IsGroup, users, groups)
-
-	p, err := o.ToPrinter(message)
-	if err != nil {
-		return err
+	clusterRole := &rbacv1.ClusterRole{
```

Comment:
Hmm... I struggled with that thought, but ok. Lemme open a apiserver PR doing that too, and this will be a fallback in case we don't have that.

Reply:
https://github.com/openshift/cluster-kube-apiserver-operator/tree/master/manifests could be a good spot for these as SCCs are already there

Reply:
Hmm... I went with https://github.com/openshift/openshift-apiserver/pull/108 since that seemed the most natural to me, scc belonging to openshift-apiserver.

Reply:
Spoke with Standa on slack, I'll move this to kas-o, since SCCs CRDs are created there, so it's logical continuation.

Reply:
https://github.com/openshift/cluster-kube-apiserver-operator/pull/856